lib/osf-components/addon/components/storage-provider-manager/storage-manager/component.ts

```diff
@@ -132,8 +146,17 @@ export default class StorageManager extends Component<Args> {
     @waitFor
     async getRootFolder() {
         if (this.args.provider) {
-            this.storageProvider = this.args.provider;
-            const providerFile = getStorageProviderFile(this.currentUser, this.storageProvider);
+            this.externalStorageService = this.args.provider;
+            let providerFile;
+            if(this.features.isEnabled('gravy_waffle') && this.externalStorageService.name !== 'osfstorage') {
```

Comment:
May be a good idea to use some `defaultProvider` from a config in the future. Hardcoding this seems like it would be problematic if people are trying to use their own OSF instance